sdk/digitaltwins/Azure.DigitalTwins.Core/samples/DigitalTwinsClientSample/ComponentSamples.cs

```diff
@@ -84,7 +83,7 @@ public async Task RunSamplesAsync(DigitalTwinsClient client)
 
                 Console.WriteLine($"Retrieved and deserialized digital twin {basicDt.Id}  with ETag {basicDt.ETag} " +
                     $"and Prop1 '{basicDt.CustomProperties["Prop1"]}', Prop2 '{basicDt.CustomProperties["Prop2"]}', " +
-                    $"ComponentProp1 '{component1["ComponentProp1"]}', ComponentProp2 '{component1["ComponentProp2"]}'");
+                    $"ComponentProp1 '{component1["ComponentProp1"]}', ComponentProp2 {component1["ComponentProp2"]}");
```

Comment:
Removing the quotes from the console output format, since this property is now an integer

Reply:
The quote was to designate what was in the value, and what was not.

If quotation marks are mislead, perhaps we can use something else? Maybe square brackets?

Reply:
I see, how about:
```
Retrieved and deserialized digital twin sampleTwin9433  with ETag b92d0ace-61e1-4734-ac86-94228f4c08ee and properties:
 Prop1: 'Value1',
 Prop2: 'Value2,'
 ComponentProp1: 'ComponentValue1',
 ComponentProp2: 123
``